### PR TITLE
Update tokens-eth.json

### PR DESCRIPTION
**Pull request by dougsandlin**
Updated information for the ATLANT token (ATL), specifically to link to our new logo, and also to add some social media links.

### CODE REVIEW

File: tokens/tokens-eth.json [hunk 1]
```diff
@@ -15247,31 +15248,6 @@
 			"youtube": ""
 		}
 	},
-	{
```
Comment (gamalielhere) [inline tokens/tokens-eth.json]: Can you merge the master branch to this pr so you don't remove this token?

Comment (dougsandlin) [reply tokens/tokens-eth.json]: My apologies, but I am not sure what else to do to make that happen. I definitely don't want to remove the token. I will research how to do this, but any advice or assistance you can offer would be very much appreciated. Thank you.

**Comment by dougsandlin** [comment]
After fairly extensive review, I'm not sure what to do to merge the Master Branch to this pull request, per the comment above, or if I have the authority to do so. If anyone can help with this, or provide needed input, that would be much appreciated. Thank you.

**Comment by dougsandlin** [comment]
I don't have write access, but there are no conflicts with the base branch, per the message above. 

Can someone with write access please merge this pull request? Thank you.

**Comment by gamalielhere** [comment]
@dougsandlin updated it for you

**Comment by dougsandlin** [comment]
@gamalielhere - Thanks very much.

**Pull request merged**